src/app/components/button/button.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { ButtonItem } from 'src/app/interfaces/button.interface';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent  {
+  @Input() content: ButtonItem = {name: 'Female'}
```

Comment:
It's no necessary to create a interface for a single string.

```suggestion
  @Input() content: string = 'Female';
```